src/test/java/org/apache/accumulo/testing/continous/KeySerializationComparisonTest.java

```diff
@@ -0,0 +1,170 @@
+package org.apache.accumulo.testing.continous;
+
+import java.util.List;
+import java.util.Random;
+
+import org.apache.accumulo.core.data.Key;
+import org.apache.accumulo.core.security.ColumnVisibility;
+import org.apache.accumulo.core.util.FastFormat;
+import org.apache.accumulo.testing.continuous.ContinuousIngest;
+import org.apache.accumulo.testing.continuous.TestKey;
+import org.apache.commons.lang3.time.StopWatch;
+import org.apache.hadoop.io.WritableUtils;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class KeySerializationComparisonTest {
+
+  private static List<ColumnVisibility> visibilities;
+  private static final byte[] EMPTY_BYTES = new byte[0];
+  private static final long MEGABYTE = 10 * 1024L * 1024L;
+  Random random;
+
+  @Before
+  public void setup() {
+    random = new Random(System.currentTimeMillis());
+    visibilities = ContinuousIngest.parseVisibilities("IMATEST");
+  }
+
+  public static byte[] genCol(long cfInt) {
```

Comment:
Could call `ContinuousIngest.genCol()` instead and avoid duplicating code.  Could statically import that function.

Reply:
ha, yeah I was copying code around. Thanks for pointing this out!